sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -59,29 +91,31 @@ def _populate_metric_results(self, response):
     # Get the tentative/committed versions of every metric together.
     metrics_by_name = defaultdict(lambda: {})
     for metric in user_metrics:
+      if (metric.name.name.endswith('(DIST)') or
+          metric.name.name.endswith('[MIN]') or
+          metric.name.name.endswith('[MAX]') or
+          metric.name.name.endswith('[MEAN]') or
+          metric.name.name.endswith('[COUNT]')):
+        warn('Distribution metrics will be ignored in the MetricsResult.query'
```

Comment:
It will only be shown once
http://stackoverflow.com/questions/22661745/why-only-one-warning-in-a-loop